tests/rss/test_worklows.py

```diff
@@ -131,6 +116,84 @@ def mock_do_RSS_iterations(input: Dict[str, Optional[Any]] = {'test_error': None
                       max_iteration_number: int = 9,
                       **fit_kwargs,):
 
+    if input['test_error'] is not None and input['test_error'] > stop_criterion and input['current_iter'] < max_iteration_number:
+        if input['kt'] > 0.15:
+            kt = input['kt'] - 0.1
+        else:
+            kt = 0.1
+        print('kt:', kt)
+        current_iter = input['current_iter'] + 1
+        print('Current iter index:', current_iter)
+        print(f'The error of {current_iter}th iteration:', input['test_error'])
```

Comment:
Is this printing strictly needed? Can we use logging instead or remove it?

Reply:
I think it makes sense to print it out. Yes, it can serve as a log output.